microprofile/microprofile-opentracing-1.3/src/test/java/org/wildfly/swarm/ts/microprofile/opentracing/v13/MPOpenTracing13Test.java

```diff
@@ -0,0 +1,202 @@
+import static org.awaitility.Awaitility.await;
+
+@RunWith(Arquillian.class)
+@DefaultDeployment
+public class MPOpenTracing13Test {
+    private static final String PREFIX = "org.wildfly.swarm.ts.microprofile.opentracing.v13.";
+
+    private static final String GET_PREFIX = "GET:" + PREFIX;
+
+    private static final String URL_REST = "http://localhost:8080/rest/";
+
+    private ImmutableMap<Object, Object> URL_TO_RESPONSES = ImmutableMap.builder()
+            .put("tracedlogged", "Hello from tracedLoggedMethod")
+            .put("traced", "Hello from tracedMethod")
+            .put("logged", "Hello from loggedMethod")
+            .put("nottracednotlogged", "Hello from notTracedNotLoggedMethod")
+
+            .put("named/true", "Hello from tracedNamedTrue")
+            .put("true", "Hello from tracedTrue")
+            .put("named/false", "Hello from tracedNamedFalse")
+            .put("false", "Hello from tracedFalse")
+
+            .put("test/1/hello", "Hello from twoWildcard: 1, hello")
+            .build();
+
+    private List<Trace> EXPECTED_TRACES = Arrays.asList(
+            // traced and tracer.span.logging combinations
+            new Trace("Resource.tracedLoggedMethod", null, PREFIX + "TracedService.tracedLoggedMethod", "tracer: tracedLoggedMethod"),
+            new Trace("Resource.tracedMethod", null, PREFIX + "TracedService.tracedMethod", null),
+            new Trace("Resource.loggedMethod", "tracer: loggedMethod"),
+            new Trace("Resource.notTracedNotLoggedMethod"),
+            // operationName with on/off combination
+            new Trace("Resource.tracedNamedTrue", null, "operationName-should-appear-tracedNamedTrue", "tracedNamedTrue"),
+            new Trace("Resource.tracedTrue", null, PREFIX + "TracedService.tracedTrue", "tracedTrue"),
+            new Trace("Resource.tracedNamedFalse", "tracedNamedFalse"),
+            new Trace("Resource.tracedFalse", "tracedFalse"),
+            // two wild cards
+            new Trace("Resource.twoWildcard", null, PREFIX + "TracedService.twoWildcard", "twoWildcard: 1, hello")
+    );
+
+    @ClassRule
+    public static Docker jaegerContainer = new Docker("jaeger", "jaegertracing/all-in-one:latest")
+            .waitForLogLine("\"Health Check state change\",\"status\":\"ready\"")
+            // https://www.jaegertracing.io/docs/1.11/getting-started/
+            .port("5775:5775/udp")
+            .port("6831:6831/udp")
+            .port("6832:6832/udp")
+            .port("5778:5778")
+            .port("16686:16686")
+            .port("14250:14250")
+            .port("14267:14267")
+            .port("14268:14268")
+            .port("9411:9411")
+            .envVar("COLLECTOR_ZIPKIN_HTTP_PORT", "9411")
+            .cmdArg("--reporter.grpc.host-port=localhost:14250");
+
+    @Test
+    @InSequence(1)
+    @RunAsClient
+    public void applicationRequest() {
+        URL_TO_RESPONSES.forEach((k, v) -> {
```

Comment:
`k` -> `url`, `v` -> `expectedResponse`

Reply:
What do you prefer? pair of (k, v) on map is quite straightforward, I am open to any suggestion..
URL_TO_RESPONSES.entrySet().forEach(e -> assertThat(e.getKey(), e.getValue())); can be better..?

Reply:
I'm just asking for rename of the variables :-)